doc/manual/command-ref/conf-file.xml

```diff
@@ -468,6 +468,29 @@ builtins.fetchurl {
     this option to <literal>true</literal>.</para></listitem>
   </varlistentry>
 
+  <varlistentry xml:id="conf-log-all-ifd"><term><literal>log-all-ifd</literal></term>
+
+    <listitem>
+
+      <para>Default: <literal>false</literal>.</para>
+
+      <para>If set to <literal>true</literal>, the Nix evaluator will
+      emit log messages for all imports from derivation at the 'info'
+      log level.  Example output with and without this option is:</para>
+
+      <informalexample><screen>
+$ nix-build -E 'with import <nixpkgs> {}; haskellPackages.callHackage "turtle" "1.5.15" {}' -o turtle --option log-all-ifd true
+/nix/store/c1jyh069rg3cdlpx3dj0466qg39gq4dl-nixos-20.03pre202088.e89b21504f3/nixos/pkgs/development/haskell-modules/make-package-set.nix:83:54 importing from derivation /nix/store/vcj52jqhnbhqg44n4myfbwplax3q89f6-cabal2nix-turtle-1.5.15.drv via scopedImport
+/nix/store/nb8qg80a5vv2gj145h20z90ynzlibbrq-turtle-1.5.15
+
+$ nix-build -E 'with import <nixpkgs> {}; haskellPackages.callHackage "turtle" "1.5.15" {}' -o turtle
+/nix/store/nb8qg80a5vv2gj145h20z90ynzlibbrq-turtle-1.5.15
```

Comment:
```suggestion
```

let's delete this. We don't need to show what the absence of the message looks like :)